WordPress/Classes/ViewRelated/Blog/WPStyleGuide+Blog.swift

```diff
@@ -11,7 +11,7 @@ extension WPStyleGuide {
         cell.detailTextLabel?.sizeToFit()
         cell.detailTextLabel?.textColor = .textSubtle
 
-        cell.imageView?.layer.borderColor = UIColor.white.cgColor
+        cell.imageView?.layer.borderColor = UIColor.divider.cgColor
         cell.imageView?.layer.borderWidth = 1
```

Comment:
I think this `borderWidth` needs to be `hairlineBorderWidth` (so it adjusts with screen size). It looks awfully thick on an iPad.

<img width="194" alt="Screen Shot 2020-12-03 at 5 11 21 PM" src="https://user-images.githubusercontent.com/1816888/101105356-9797d080-358a-11eb-8b82-032e70fc3c5b.png">